File: codigos/1 - estrutura sequencial/1012 - Area/Main.py
from math import pow

A, B, C = map(float, input().split())

area_triangulo: float = A*C / 2
area_circulo: float = 3.14159 * pow(C, 2)
area_trapezio: float = (A+B) * C / 2
area_quadrado: float = pow(B,2)
area_retangulo: float = A*B

print(f"TRIANGULO: {area_triangulo:.3f}")
print(f"CIRCULO: {area_circulo:.3f}")
print(f"TRAPEZIO: {area_trapezio:.3f}")
print(f"QUADRADO: {area_quadrado:.3f}")
print(f"RETANGULO: {area_retangulo:.3f}")

File: codigos/1 - estrutura sequencial/1012 - Area/Main.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    private static String area_triangulo(double A, double C){
        return String.format("%.3f", A*C/2);
    }
    private static String area_circulo(double C){
        return String.format("%.3f", 3.14159*C*C);
    }
    private static String area_trapezio(double A, double B, double C){
        return String.format("%.3f", (A+B)*C/2);
    }
    private static String area_retangulo(double A, double B){
        return String.format("%.3f", A*B);
    }
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner prompt = new Scanner(System.in);
        double A = prompt.nextDouble();
        double B = prompt.nextDouble();
        double C = prompt.nextDouble();
        System.out.println("TRIANGULO: " + area_triangulo(A, C));
        System.out.println("CIRCULO: " + area_circulo(C));
        System.out.println("TRAPEZIO: " + area_trapezio(A, B, C));
        System.out.println("QUADRADO: " + area_retangulo(B, B));
        System.out.println("RETANGULO: " + area_retangulo(A, B));
    }
}